client/src/components/Atoms/Table.tsx

```diff
@@ -135,4 +135,55 @@ export const sortByRank = <T extends object>(rank: (data: T) => number) =>
     [rank]
   )
 
+// FlexTable uses flexbox styling to compute thead/tbody height.
+// This allows us to create a scrollable table body when there's overflow.
+// Columns will all have the same width by default.
+// Based on https://stackoverflow.com/a/29512692/1472662
+// eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
+export const FlexTable = styled(({ scrollable, ...props }) => (
+  <HTMLTable {...props} />
+))`
+  display: flex;
+  flex-direction: column;
+  box-shadow: 0 0 0 1px rgb(16 22 26 / 15%); /* Copied from Blueprint */
+  width: 100%;
+
+  thead {
+    flex: 0 0 auto;
+    box-shadow: inset 0 -1px 0 0 rgb(16 22 26 / 15%); /* Copied from Blueprint */
+    width: 100%;
+  }
+
+  thead tr {
+    /* Remove approx width of scrollbar so headers line up with columns */
```

Comment:
Out of curiosity, does this approach work across different OSs and browsers?

Reply:
Tried a couple browsers and it was subtly different. I found a new solution where you add a hidden scrollbar to the header row instead: f198cbc9

Reply:
Huh, cool!